ui/tsconfig.json

```diff
@@ -8,12 +8,8 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
-    "lib": [
-      "es2016",
-      "es2018.promise",
```

Comment:
Since we are no longer compiling down to es5, we don't need to explicitly enable these features.